tests/test_helpers.py

```diff
@@ -29,6 +29,7 @@
 
 from xdg import BaseDirectory
 
+from fades import HTTP_STATUS_NOT_FOUND, HTTP_STATUS_OK
 from fades import helpers
 from fades import parsing
```

Comment:
I wonder why don't we have all this imports in the same line...

Reply:
done